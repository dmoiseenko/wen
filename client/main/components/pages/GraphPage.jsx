import React from 'react';

import Import from '../common/Import';


export default function GraphPage() {
  return (
    <div className="container">
      <div className="content">
        <h2>Touch And Drag</h2>
      </div>
      <Import load={import('../graph/Graph')} />
    </div>
  );
}
