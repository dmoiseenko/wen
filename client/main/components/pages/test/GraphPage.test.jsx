import React from 'react';
import { shallow } from 'enzyme';

import GraphPage from '../GraphPage';


it('should render', () => {
  const wrapper = shallow(
    <GraphPage />
  );

  expect(wrapper.node).toMatchSnapshot();
});
