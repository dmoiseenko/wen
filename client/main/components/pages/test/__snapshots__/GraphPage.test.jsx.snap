// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render 1`] = `
<div
  className="container"
>
  <div
    className="content"
  >
    <h2>
      Touch And Drag
    </h2>
  </div>
  <Import
    load={Promise {}}
  />
</div>
`;
